observability/WFLY-20296-add-support-for-more-opentelemetry-config-options.adoc

```diff
@@ -0,0 +1,154 @@
+---
+categories:
+- observability
+stability-level: community
+# Specify the Feature Development tracker issue for the feature.
+# This must be an issue tracked in https://github.com/orgs/wildfly/projects/7/views/1.
+# To create a Feature Development tracker issue, go to  https://github.com/wildfly/wildfly-proposals/issues/new/choose
+# and select 'Feature Development'
+issue: https://github.com/wildfly/wildfly-proposals/issues/683
+feature-team:
+ developer: jasondlee
+ sme:
+  -
+ outside-perspective:
+  - TomasHofman
+promotes:
+promoted-by:
+---
+= WFLY-20296 - Add support for more OpenTelemetry configuration options
+:author:            Jason Lee
+:email:             jasondlee@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+While every aspect of OpenTelemetry is technically configurable via MP Telemetry and MP Config, there have been several requests for more direct support for the various options in the management model. The RFE will identify and implement the options most likely to be of general interest to WildFly users.
+
+=== User Stories
+
+- As an application developer, I would like greater control in how OpenTelemetry is configured in the server without having to depend on an external library (e.g., MicroProfile Config).
+
+== Issue Metadata
+
+- https://issues.redhat.com/browse/WFLY-20296[]
+
+=== Related Issues
+
+- https://issues.redhat.com/browse/WFLY-18290[]
+- https://issues.redhat.com/browse/WFLY-19919[]
+- https://issues.redhat.com/browse/JBEAP-27199[]
+
+=== Affected Projects or Components
+
+This change will affect only the OpenTelemetry subsystem
+
+=== Other Interested Projects
+
+=== Relevant Installation Types
+
+* Traditional standalone server (unzipped or provisioned by Galleon)
+* Managed domain
+* OpenShift Source-to-Image (S2I)
+* Bootable jar
+
+== Requirements
+
+* Additional configuration options should include:
+|===
+|Signal type|Name|OpenTelemetry Property|Description|Attribute Name|Default|Notes
+|General|TLS server certificate|`otel.exporter.otlp. certificate`|The path to the server certificate on disk|TBD||_TLS props are pending a discussion with Elytron_
+||TLS client certificate|`otel.exporter.otlp.client. certificate`|The path to the client certificate on disk|TBD||
+||Client key|`otel.exporter.otlp.client. key`|The path to the key on disk|TBD||
+||Compression|`otel.exporter.otlp. compression`|The compression type to use on OTLP trace, metric, and log requests.|`compression`|null|Only `gzip` supported
+|Traces|Enabled|`otel.traces.exporter`|Are traces enabled?|`traces-enabled`|true|If true, the value is set to `otlp`. If not, the value is set to `none`.
```

Comment:
What's the plan for when we decide to support other exporters?

I can imagine one; I just want to make sure you have too, as you'll be living with this for many years. :-)

Same applies to Metrics|Enabled